bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -484,4 +484,438 @@
+		<item-type>Number:Power</item-type>
+		<label>DC Power PV</label>
+		<description>Current power from all solar panels</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc1-voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC1 Voltage</label>
+		<description>Current voltage from solar panels, DC1</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc1-current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>DC1 Current</label>
+		<description>Current current from solar panels, DC1</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc1-power">
+		<item-type>Number:Power</item-type>
+		<label>DC1 Power</label>
+		<description>Current power from solar panels, DC1</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc2-voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC2 Voltage</label>
+		<description>Current voltage from solar panels, DC2</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc2-current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>DC2 Current</label>
+		<description>Current current from solar panels, DC2</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc2-power">
+		<item-type>Number:Power</item-type>
+		<label>DC2 Power</label>
+		<description>Current power from solar panels, DC2</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc3-voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>DC3 Voltage</label>
+		<description>Current voltage from solar panels, DC3</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc3-current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>DC3 Current</label>
+		<description>Current current from solar panels, DC3</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-dc3-power">
+		<item-type>Number:Power</item-type>
+		<label>DC3 Power</label>
+		<description>Current power from solar panels, DC3</description>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="device-local-akt-home-consumption-solar">
+		<item-type>Number:Power</item-type>
+		<label>Actual Home Consumption Solar</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions